error:
       … from call site
         at /pwd/lang/eval-fail-mutual-recursion.nix:40:1:
           39| in
           40| throwAfterA true 10
             | ^
           41|

       … while calling 'throwAfterA'
         at /pwd/lang/eval-fail-mutual-recursion.nix:32:14:
           31|   throwAfterA =
           32|     recurse: n:
             |              ^
           33|     if n > 0 then

       … from call site
         at /pwd/lang/eval-fail-mutual-recursion.nix:34:7:
           33|     if n > 0 then
           34|       throwAfterA recurse (n - 1)
             |       ^
           35|     else if recurse then

       (19 duplicate frames omitted)

       … from call site
         at /pwd/lang/eval-fail-mutual-recursion.nix:36:7:
           35|     else if recurse then
           36|       throwAfterB true 10
             |       ^
           37|     else

       … while calling 'throwAfterB'
         at /pwd/lang/eval-fail-mutual-recursion.nix:23:14:
           22|   throwAfterB =
           23|     recurse: n:
             |              ^
           24|     if n > 0 then

       … from call site
         at /pwd/lang/eval-fail-mutual-recursion.nix:25:7:
           24|     if n > 0 then
           25|       throwAfterB recurse (n - 1)
             |       ^
           26|     else if recurse then

       (19 duplicate frames omitted)

       … from call site
         at /pwd/lang/eval-fail-mutual-recursion.nix:27:7:
           26|     else if recurse then
           27|       throwAfterA false 10
             |       ^
           28|     else

       (21 duplicate frames omitted)

       … while calling the 'throw' builtin
         at /pwd/lang/eval-fail-mutual-recursion.nix:38:7:
           37|     else
           38|       throw "Uh oh!";
             |       ^
           39| in

       error: Uh oh!
